ENIGMAsystem/SHELL/Platforms/Cocoa/CocoaMain.cpp

```diff
@@ -25,6 +25,7 @@
 #include <limits.h>
 #include "CocoaMain.h"
 #include "ObjectiveC.h"
+#include <string.h>
```

Comment:
In C++ sources, use `<cstring>` instead.

Reply:
I honestly didn't know that there was a c++ version of that header. Thanks for the info.